compiler/imgdata2hdf5/imgdata2hdf5.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+import h5py as h5
+import numpy as np
+import argparse
+import glob
+import os
+
+parser = argparse.ArgumentParser()
+parser.add_argument(
+    "-l",
+    "--data_list",
+    type=str,
+    help=
+    "Path to the text file which lists the absolute paths of the raw image data files to be converted.",
+    required=True)
+parser.add_argument(
+    "-o", "--output_path", type=str, help="Path to the output hdf5 file.", required=True)
+
+args = parser.parse_args()
+data_list = args.data_list
+output_path = args.output_path
+
+# Create h5 file
+h5_file = h5.File(output_path, 'w')
+group = h5_file.create_group("value")
+# We assume the raw input data have the correct type/shape for the corresponding model
+# If this flag is set in the hdf5 file, record-minmax will skip type/shape check
+group.attrs['rawData'] = '1'
+
+if os.path.isfile(data_list) == False:
+    raise SystemExit("No such file. " + data_list)
+
+# Data list
+datalist = []
+with open(data_list, 'r') as f:
+    lines = f.readlines()
+    for line in lines:
+        filename = line.rstrip()
+        if os.path.isfile(filename):
+            datalist.append(filename)
+        else:
+            raise SystemExit("No such file. " + filename)
```

Comment:
this will stop when there is an empty line or only with spaces that users can use as some delimiter

Reply:
I changed the code to support skipping of empty lines